core/exception/src/main/java/uk/gov/gchq/gaffer/core/exception/Status.java

```diff
@@ -89,10 +90,10 @@
 
     public static Status fromStatusCode(final int statusCode) {
         return Arrays.asList(values())
-                     .stream()
```

Comment:
I've noted on gh-890 that we need to make sure we're all using the same formatting settings to avoid changes like this creeping into pull requests too often. We need to agree and publish this somewhere and make sure we're all following it.